README.md

```diff
@@ -71,69 +146,89 @@ az ad sp create-for-rbac --name "mySQLServer" --role contributor \
-      with:
-        creds: ${{ secrets.AZURE_CREDENTIALS }}
-    - uses: azure/sql-action@v1
+    - uses: azure/sql-action@v1.2
       with:        
-        server-name: REPLACE_THIS_WITH_YOUR_SQL_SERVER_NAME
         connection-string: ${{ secrets.AZURE_SQL_CONNECTION_STRING }}
         project-file: './Database.sqlproj'
-        build-arguments: '-c Release'                 # Optional arguments passed to dotnet build
-        arguments: '/p:DropObjectsNotInSource=true'   # Optional parameters for SqlPackage Publish
 ```
+3. Place the connection string from the Azure Portal in GitHub secrets as `AZURE_SQL_CONNECTION_STRING`. Connection string format is: `Server=<server.database.windows.net>;User ID=<user>;Password=<password>;Initial Catalog=<database>`.
+1. Copy the below SQL project template and paste the content in your project repository as `Database.sqlproj`.
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build">
+  <Sdk Name="Microsoft.Build.Sql" Version="0.1.3-preview" />
+  <PropertyGroup>
+    <Name>reactions</Name>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.SqlAzureV12DatabaseSchemaProvider</DSP>
+    <ModelCollation>1033, CI</ModelCollation>
+  </PropertyGroup>
+</Project>
+```
+5. Place any additional SQL object definitions in the project folder or in subfolders.  An example table to get you started is:
+```sql
+CREATE TABLE [dbo].[Product](
+	[ProductID] [int] IDENTITY(1,1) PRIMARY KEY NOT NULL,
+	[Name] [nvarchar](100) NOT NULL,
+	[ProductNumber] [nvarchar](25) NOT NULL,
+	[Color] [nvarchar](15) NULL,
+	[StandardCost] [money] NOT NULL,
+	[ListPrice] [money] NOT NULL,
+	[Size] [nvarchar](5) NULL,
+	[Weight] [decimal](8, 2) NULL,
+	[ProductCategoryID] [int] NULL,
+	[ProductModelID] [int] NULL,
+	[ModifiedDate] [datetime] NOT NULL
+)
+```
+6. Commit and push your project to GitHub repository, you should see a new GitHub Action initiated in **Actions** tab.
+7. For further use of SQL projects in VS Code and Azure Data Studio, check out [http://aka.ms/azuredatastudio-sqlprojects](http://aka.ms/azuredatastudio-sqlprojects) for more information.
 
-**Note:**
-The database project must use the [Microsoft.Build.Sql](https://www.nuget.org/packages/microsoft.build.sql/) SDK.
 
-### Sample workflow to deploy a DACPAC to an Azure SQL database
+### Create Azure SQL Database + 
 
+1. Create a dacpac from an existing SQL database with either [SSMS](https://docs.microsoft.com/sql/relational-databases/data-tier-applications/extract-a-dac-from-a-database), [Azure Data Studio](https://docs.microsoft.com/sql/azure-data-studio/extensions/sql-server-dacpac-extension) or [SqlPackage CLI](https://docs.microsoft.com/sql/tools/sqlpackage/sqlpackage-extract).  Place the dacpac file at the root of your repository.
+1. Follow the tutorial [Azure SQL Quickstart to create a single database](https://docs.microsoft.com/azure/azure-sql/database/single-database-create-quickstart?tabs=azure-portal#create-a-single-database)

```

Comment:
I know markdown rendering handles this for free, but may as well still increment the list ordinal.